src/helpers/MeasureBundleHelpers.ts

```diff
@@ -5,6 +5,7 @@ import { UnexpectedProperty, UnexpectedResource } from '../types/errors/CustomEr
 import { getMissingDependentValuesets } from '../execution/ValueSetHelper';
 import { ValueSetResolver } from '../execution/ValueSetResolver';
 import { ExtractedLibrary } from '../types/CQLTypes';
+import { MeasureBundleHelpers } from '..';
```

Comment:
Looks like this file is importing itself? You should be able to just call the local function below rather than importing it and using this as a prefix

Reply:
this was vscode not me I swear... apologies for letting this get pushed this is so embarrassing